### ISSUE TITLE
Parts of editor not visible when adding a TextEdit control

### ISSUE THREAD

**Issue by sharedrory** [godotengine/godot#49958]
### Godot version

3.3.2.stable.official

### System information

Windows 7 GLES2 Intel(R) Q45/Q43 Express Chipset Batching On

### Issue description

I created a new project and was making a UI scene. I added some containers and went to add a `TextEdit`. When I added it, suddenly parts of the editor got their colors turned to the default 2d scene background color. Ctrl + Z (to remove TextEdit) makes it all good again.

BEFORE:
![image](https://user-images.githubusercontent.com/85806407/123562535-45640a00-d77d-11eb-8f97-b54367b030bb.png)

AFTER:
![image](https://user-images.githubusercontent.com/85806407/123562550-590f7080-d77d-11eb-8dda-31cdf3398587.png)

Menus are still clickable just the toolbar buttons aren't visible.
When it first happened, I tried to quit to project list and reopen project. No luck.

I did some testing afterwards and found that zooming in to the UI scene fixes it.

Using just the mouse wheel, the zoom level that is ok:
![image](https://user-images.githubusercontent.com/85806407/123562605-bdcacb00-d77d-11eb-8043-2493e1aa0bf2.png)

I also tried to close editor and restart it. No luck.

I did some more testing and found out the zoom may not have exactly fixed it.
Basically, if a certain part of the left side of the scene is visible, it will cause the issue and it only happens in the UI scene.
___
Console:
![image](https://user-images.githubusercontent.com/85806407/123562740-9f190400-d77e-11eb-9119-e2d8969e65a8.png)
That error is always visible for me tho since the beginning when I started using godot. It is what prevents me from making 3d games. If it helps, godot says my GPU drivers don't support GLES3.

### Steps to reproduce

1. Create a new GLES2 Project
2. Add a 2d scene and save it
3. Add a user interface scene with a HBoxContainer containing a VBoxContainer
4. Add a TextEdit to the VBoxContainer

**Comment by Calinou** [comment]
Can you reproduce this after disabling batching for both the project and editor in the Project ?Settings

**Comment by sharedrory** [comment]
Yep, still reproducible. I just noticed, the TextEdit isn't actually visible. Also, moving the scene so that (0, 0) isn't visible fixes it.
